R/install-remote.R

```diff
@@ -60,20 +60,8 @@ install_remote <- function(remote, ..., force = FALSE, quiet = FALSE,
           repos = repos, type = type)
 }
 
-try_install_remote <- function(..., quiet = FALSE) {
```

Comment:
I removed this mainly because it was useful for running revdep checks, it was added in https://github.com/r-lib/devtools/pull/1284 and we no longer use devtools to do this.

It also makes the user experience when installing a single repository worse, as there is no longer an explicit error on failure to download the package.